### PR TITLE
act_dr6_lenslike code from PyPI

### PR DESCRIPTION
**Pull request by itrharrison**
Rather than including the `act_dr6_lenslike` python code directly, we instead assume it has already been installed (i.e. from PyPI).

### CODE REVIEW

**Comment by itrharrison** [comment]
Ack. I didn't mean to open this with `joezuntz:main` as the target. I don't think this is appropriate for merging yet as the module install happens from test-PyPI.

**Comment by itrharrison** [comment]
@joezuntz this should be ready to go now.

File: likelihood/act-dr6-lens/act_dr6_lenslike_interface.py [hunk 1]
```diff
@@ -27,23 +66,34 @@ def execute(block, config):
...
+        cl_kk = act_dr6_lenslike.pp_to_kk(cl_pp, ell)
 
     # Then call the act code
-    lnlike, bclkk = act_dr6_lenslike.generic_lnlike(data_dict,ell, cl_kk, ell, cl_tt, cl_ee, cl_te, cl_bb)
+    lnlike = act_dr6_lenslike.generic_lnlike(data_dict,ell, cl_kk, ell, cl_tt, cl_ee, cl_te, cl_bb, data_dict['trim_lmax'])
     block[names.likelihoods, 'act_dr6_lens_like'] = lnlike
 
     if not data_dict['cosmosis_like_only']:
-        block[names.data_vector, 'act_dr6_lens_theory'] = bclkk
+        # block[names.data_vector, 'act_dr6_lens_theory'] = bclkk
```
Comment (joezuntz) [inline likelihood/act-dr6-lens/act_dr6_lenslike_interface.py:96]: It's useful to save the theory, though I know that would require changes to the library.  I'll open an issue there.

File: likelihood/act-dr6-lens/act_dr6_lenslike_interface.py [hunk 2]
```diff
@@ -27,23 +66,34 @@ def execute(block, config):
...
-    # Slightly different normalization here
+    # Slightly different normalization here
     cl_pp = block[names.cmb_cl, 'pp'] / f1
 
-    cl_kk = act_dr6_lenslike.pp_to_kk(cl_pp, ell)
+    if data_dict['varying_cmb_alens']:
+    # if block.has_value(cosmo, 'A_lens'):
+        cl_pp /= block[cosmo, "A_lens"]
+
+    if data_dict['limber']:
```
Comment (joezuntz) [inline likelihood/act-dr6-lens/act_dr6_lenslike_interface.py:86]: Want to make sure this error is raised in the setup function instead of execute.

**Review by joezuntz — CHANGES_REQUESTED** [review]

File: likelihood/act-dr6-lens/act_dr6_lenslike_interface.py (hunk 2 above)
Comment (joezuntz) [reply likelihood/act-dr6-lens/act_dr6_lenslike_interface.py:88]: Sorry - I think this was ambiguous - I meant the fact that the limber option is not implemented, not the A_lens bit.

Comment (joezuntz) [reply likelihood/act-dr6-lens/act_dr6_lenslike_interface.py:88]: Sorry - I think this was ambiguous - I meant the fact that the limber option is not implemented, not the A_lens bit.

File: likelihood/act-dr6-lens/module.yaml [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+explanation: |
+    "This is the lensing likelihood from the ACT DR6 data release. What is supplied here is only the CosmoSIS interface
+    to the standalone act_dr6_lenslike python module. This should be obtainable through running: pip install act_dr6_lenslike."
+
+# List of parameters that can go in the params.ini file in the section for this module    
+params:
+    data_directory:
+        meaning: "Directory to search for data"
+        type: "[module dir]/data"
```
Comment (joezuntz) [inline likelihood/act-dr6-lens/module.yaml:26]: I think this is the default not the type, which is str

**Review by joezuntz — CHANGES_REQUESTED** [review]

**Comment by joezuntz** [comment]
Just one quick change needed.

**Pull request merged**